client/src/components/EditProfile/EditForm/EditForm.tsx

```diff
@@ -80,17 +83,56 @@ const formValidation = Yup.object().shape({
 const EditForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
   const options = ['Male', 'Female', 'Other'];
+  const [isAvailable, setIsAvailable] = useState(false);
+
+  const daysOptions = ['Monday', 'Tuesday', 'Wednesday', 'Thrusday', 'Friday', 'Saturday', 'Sunday'];
+
+  const handleSwitch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsAvailable(event.target.checked);
+  };
+
   return (
     <Box>
       <Formik initialValues={initialFormState} validationSchema={formValidation} onSubmit={handleSubmit}>
         {({ handleSubmit }) => (
           <Form onSubmit={handleSubmit}>
             <Grid container alignItems="center" spacing={2}>
               <Grid item xs={12}>
-                <Typography variant="h6" align="center">
+                <Typography variant="h5" align="center">
                   Edit Profile
                 </Typography>
               </Grid>
+              <Grid item xs={12} sm={3}>
+                <Typography variant="body1" color="textPrimary" className={classes.typography}>
+                  {isAvailable ? `I 'm available` : `I 'm not available`}
+                </Typography>
+              </Grid>
+              <Grid item xs={12} sm={8}>
+                <Switch color="error" onChange={handleSwitch} />
+              </Grid>
+              <Grid item xs={12} sm={3}>
+                <Typography variant="body1" color="textPrimary" className={classes.typography}>
+                  Availability
+                </Typography>
+              </Grid>
+              <Grid item xs={12} sm={8}>
+                <FormGroup>
+                  <Box display="flex" flexWrap="wrap">
+                    {daysOptions.map((day, i) => (
+                      <div key={i}>
+                        {isAvailable ? (
+                          <>
+                            <Field type="checkbox" name="availability" label={day} value={day} as={Checkbox} />
```

Comment:
There are several ways to use Formik and which implementation is chosen is usually a question of preference and convention. Either way, it's not a good idea to mix and match different styles of using Formik. At [`line 97`](https://github.com/hatchways/team-reindeer/blob/fa2adca472ed7df469a09bb1f5b87379229828d6/client/src/components/EditProfile/EditForm/EditForm.tsx#L97), we use the render function to access Formik's context in the UI so we should stick to that instead of additionally using other Formik components such as `Form` or `Field`. 

With that in mind, we should use a mui checkbox ([docs](https://mui.com/components/checkboxes/)) here instead of `Field`, that way you can change it's color just by passing a prop to it as we always do! Anything you would need to pass to a mui checkbox in order to integrate it with Formik would be available in the props of this render function (e.g. you can modify `line 97` to access them as such):
 ```
        {({ values, touched, errors, handleChange, handleSubmit, ...}) => (
```
You can take a look at the Formik docs [here](https://formik.org/docs/api/formik) to see what other props are available too 👍🏼